packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.tests.js

```diff
@@ -4335,3 +4343,16 @@ QUnit.module('integration of dx button components via dialogTrigger', moduleConf
         });
     });
 });
+
+QUnit.module('labelText', moduleConfig, () => {
+    QUnit.test('Dropzone default label text should be hidden on mobile devices', function(assert) {
+        if(shouldSkipOnDesktop(assert, 'should only apply on mobile devices')) {
+            return;
+        }
+
+        const $fileUploader = $('#fileuploader').dxFileUploader();
+        const $inputLabel = $fileUploader.find(`.${FILEUPLOADER_INPUT_LABEL_CLASS}`);
+
+        assert.strictEqual($inputLabel.text(), '', 'default label is hidden on mobile');
```

Comment:
[minor] In fact, the text is not hidden, but empty.
```suggestion
        assert.strictEqual($inputLabel.text(), '', 'default label text is empty on mobile');
```